kafka/avro_test.go

```diff
@@ -37,8 +38,8 @@ func TestSchemaCompatability_SigningResult_V2ToV1(t *testing.T) {
 	assert.Equal(t, v2.Status.String(), v1.Status.String())
 }
 
-//// Tests v2 consumers reading v1 messages.
-//func TestSchemaCompatability_SigningResult_V1ToV2(t *testing.T) {
+// Tests v2 consumers reading v1 messages.
```

Comment:
can we add a comment about why this is commented out?

Reply:
I'm not sure why it's commented out. I only didn't delete it because I'm not the author.

Reply:
chasing with Kit